packages/chaire-lib-backend/src/utils/processManagers/ProcessManager.ts

```diff
@@ -86,18 +97,23 @@ const startProcess = async function ({
     command,
     commandArgs,
     waitString,
-    useShell,
+    useShell = false,
     cwd,
-    attemptRestart = false
+    attemptRestart = false,
+    logFiles
 }: {
     serviceName: string;
     tagName: string;
     command: string;
     commandArgs: any;
     waitString: string;
-    useShell: boolean;
+    useShell?: boolean;
     cwd?: string;
     attemptRestart?: boolean;
+    logFiles?: {
```

Comment:
Do we realely need a struct in a struct? This make calling the function a bit more complex with useful reason.